client/question/domain/SpeechBalloonObjectFactory.js

```diff
@@ -67,6 +67,15 @@ tie.factory('SpeechBalloonObjectFactory', [
       return this._type === SPEECH_BALLOON_TYPE_FEEDBACK;
     };
 
+    /**
+     * Returns whether the code balloon is displayed on the right.
+     *
+     * @returns {boolean}
+     */
+    SpeechBalloon.prototype.isDisplayedOnRight = function() {
```

Comment:
I'd recommend calling this something like isCodeSubmission rather than isDisplayedOnRight; the latter seems more arbitrary whereas the former seems like "we want this to happen on code submission"